lib/linguist/languages.yml

```diff
@@ -2882,6 +2882,18 @@ NewLisp:
   codemirror_mode: commonlisp
   codemirror_mime_type: text/x-common-lisp
   language_id: 247
+Nextflow:
+  type: programming
+  ace_mode: text
```

Comment:
Could we use `groovy` as a substitute Ace grammar or is it too far off?

Reply:
What exactly is the impact of this? what's the `ace_mode` ?

Reply:
Ooh, yeah, forgot to give context :/
It's used to highlight code in the GitHub.com [editor](https://ace.c9.io/) (when users edit files or Gists directly in the web interface).